src/fragments/start/getting-started/reactnative/auth.mdx

```diff
@@ -0,0 +1,263 @@
+The next feature you will be adding is authentication.
+
+## Authentication with Amplify
+
+Amplify uses [Amazon Cognito](https://aws.amazon.com/cognito/) as the main authentication provider. Amazon Cognito is a robust user directory service that handles user registration, authentication, account recovery & other operations. In this tutorial, you'll learn how to add authentication to your application using Amazon Cognito and username/password login.
```

Comment:
Is there an authentication provider besides Cognito? Just thinking out loud here, but calling it the "main authentication provider" implies there may be other authentication providers

Reply:
Technically you can use Google, Facebook as a way to authenticate customers as well.

Reply:
Right, but MFA is separate from this tutorial correct? Also the current RN Authenticator does not expose MFA support (see https://github.com/aws-amplify/amplify-ui/issues/2576)

Reply:
to make things simpler, you we wouldn't want to have MFA here.

Reply:
changed the wording to avoid confusion